src/services/PushRule.js

```diff
@@ -34,6 +34,14 @@ class PushRule extends BaseService {
     return RequestHelper.get(this, `projects/${pId}/push_rule`);
   }
 
+  async update(projectId, options) {
```

Comment:
this avoid the options parameter and seems cleaner at the same time.

Reply:
I agree, my only reservation is the name, i feel like its close in meaning to edit no?